wandb/_strutils.py

```diff
@@ -38,3 +39,19 @@ def nameof(obj: Any, full: bool = True) -> str:
     falling back on the `__name__` attribute.
     """
     return getattr(obj, "__qualname__", obj.__name__) if full else obj.__name__
+
+
+def b64decodestr(s: str) -> str:
+    """Returns a decoded string from a base64-encoded string.
+
+    Convenience function for directly converting `str -> str` without converting to/from bytes.
+    """
+    return b64decode(s).decode("ascii")
```

Comment:
The "ascii" part is important, a base-64 string represents arbitrary bytes which can be interpreted differently. It should be part of the name:

```python
def b64decode_ascii(s: str) -> str:
    """Returns the decoded base64 string interpreted as ASCII."""

def b64encode_ascii(s: str) -> str:
    """Returns the base64 encoding of the string's ASCII bytes."""
```